tests/sceneManager_test.cpp

```diff
@@ -11,6 +11,68 @@ using objects::Object;
 using objects::TriangleObj;
 
 const double kSkipFreq = 1000;
+const core::Vec3 kSkipVec3(0, 0, 0);
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    std::vector<objects::EnergyCollector *> energyCollectors;
+    double energyCollectorRadius = objects::EnergyCollector(kSkipVec3).getRadius();
+    void SetUp() override
+    {
+        energyCollectors = manager.getEnergyCollectors();
+    }
+
+    bool performHitAtEnergyCollectors(const Ray &tempRay, RayHitData *hitData)
+    {
+        bool hit = false;
+        for (const auto &energyColPtr : energyCollectors)
+        {
+            if (energyColPtr->hitObject(tempRay, kSkipFreq, hitData))
+                hit = true;
+        }
+        return hit;
+    }
+
+public:
+    SceneManagerTest(){};
+};
+
+TEST_F(SceneManagerTest, PrintAllEnergyCollectorpositions)
+{
+    SetUp();
+    for (const auto &collectorPtr : energyCollectors)
+    {
+        std::cout << *collectorPtr << std::endl;
+    }
+}
+
+TEST_F(SceneManagerTest, EnergyCollectorPositionsCheck)
+{
+    SetUp();
+    ASSERT_EQ(energyCollectors.size(), constants::kCollectors) << "wrong numebr of energy collectors";
+
+    RayHitData hitData;
+    Ray tempRay(core::Vec3(0, 0, 0), core::Vec3(0, 0, 1));
```

Comment:
Why don't you name the vectors somehow so that it is clear what is their relation to the collectors?

Reply:
because Im really terrible at naming stuff in terms of relation between rays and energyCollectors in 3D space :P

Reply:
Then draw it on the paper and then name it.

Reply:
thats not the case that I dont know how corelation in 3D between Ray and colllector should look like. The problem is that I dont have idea how should  Name it.. because I cant use north/ south/ east/west or forward/ backward keywords becuse they dont provide any good information and can be misinterpretated ;)

Reply:
I am confused, you **do** or **do not** know the relation in 3D?

For the first case, how about something like this:
```
Vec3 kZero;
Ray strightUp(kZero, Vec3(0, 0, 1));
```

Next case could be:
```
Ray againstXAxis(kZero, Vec3(1, 0, 0));
```


Also note that currently this test is missing some important cases (non trivial) for this to make sense.

Reply:
I know the relation :P thanks for suggestion, done